Project.toml

```diff
@@ -1,7 +1,7 @@
 name = "NMF"
 uuid = "6ef6ca0d-6ad7-5ff6-b225-e928bfa0a386"
 license = "MIT"
-version = "1.0.2"
+version = "1.0.3"
```

Comment:
When you add a new feature, you should bump the minor version, unless the package is still at version `0.x.y`. The patch version is for bugfixes.

Reply:
You mean change the current version of NMF.jl to 1.0.3?